coregio/registry_api.py

```diff
@@ -401,3 +401,23 @@ def get_tags(self, repository: str, page_size: int = 100, limit: int = 2000) ->
         return self.get_paginated_response(
             uri, list_name="tags", page_size=page_size, limit=limit
         )
+
+    def get_layers(
+        self,
+        repositiry: str,
+        reference: str,
+    ) -> List[str]:
+        """
+        Get digests of layers in a repository by a reference (manifest digest or tag).
+
+        Args:
+            repository (str): Repository name
+            reference (str): Manifest digest or tag
+
+        Returns:
+            list: Digests of the specified image's layers
```

Comment:
Since both schema versions use a different order of layers can you please add an explanation comment to the docstring about the order and which index represents a base layer? Thanks!